lms/envs/bok_choy.py

```diff
@@ -157,6 +157,9 @@
 # Enable dashboard search for tests
 FEATURES['ENABLE_DASHBOARD_SEARCH'] = True
 
+# Enable cross-section Next button for tests
+FEATURES['ENABLE_NEXT_BUTTON_ACROSS_SECTIONS'] = True
```

Comment:
Note to self: This flag has [since been removed from master](https://github.com/edx/edx-platform/blame/master/lms/envs/bok_choy.py#L160), so should probably be removed from here too.